Makefile

```diff
@@ -84,6 +87,11 @@ $(LIBTRANSISTOR_HOME)/build/newlib/Makefile:
 $(LIBTRANSISTOR_HOME)/build/newlib/aarch64-none-switch/newlib/libc.a: $(LIBTRANSISTOR_HOME)/build/newlib/Makefile
 	$(MAKE) -C $(LIBTRANSISTOR_HOME)/build/newlib/
 
+$(LIBTRANSISTOR_HOME)/pthread/libpthread.a:
+	$(MAKE) -C $(LIBTRANSISTOR_HOME)/pthread
+
+.PHONY: $(LIBTRANSISTOR_HOME)/pthread/libpthread.a
+
```

Comment:
would prefer to avoid the use of recursive make. see comments on pthread/Makefile